examples/example_mineral.py

```diff
@@ -70,9 +70,14 @@ def run_mineral(input_filename=constants.INPUT_FILENAME,
 
     # path to save classification scores image
     scores_filename = constants.INPUT_NAME + "_scores.hdr"
-
+    
+    # path to config.ini file w/ configuration parameters
+    config_filename = os.path.join(os.path.dirname(__file__),  "..", "pycoal", "config.ini")
```

Comment:
This logic shoud not exist in the example... instead it should be pushed into the implementing Class invocation e.g.  `MineralClassification`